amundsen_application/static/js/components/TableDetail/TableIssues/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ASSOCIATION_TEXT = " is associated with this table";
+export const MAX_TEXT_LENGTH = 32; 
+export const ADDITIONAL_ISSUES_TEXT = "additional issues"; 
+export const ADDITIONAL_ISSUES_END = "associated with this table"; 
+export const ADDITIONAL_ISSUES_START = "There are"; 
```

Comment:
Seems like a lot. Can we just do something like `See all Issues` as the link?

I think Jin mentioned something about a `...` functionality to see more issues.